ui/src/templates/api/index.ts

```diff
@@ -492,3 +493,13 @@ export const installTemplate = async (orgID: string, templateUrl: string) => {
 
   return applyTemplates(params)
 }
+
+export const fetchStacks = async (orgID: string) => {
+  const resp = await getStacks({query: {orgID}})
+
+  if (resp.status >= 300) {
+    throw new Error((resp.data as PkgError).message)
+  }
+
+  return (resp.data as any).stacks
```

Comment:
does this not get types from the client?

Reply:
@ebb-tide because of the way the generated code is structured, `stacks.data` isn't typed.

```ts
interface GetStacksOKResult {
  status: 200
  headers: Headers
  data: {
    stacks?: Stack[]
  }
}
```

I did change this though to

```ts
(resp.data as {stack: InstalledStack[]}).stacks

Reply:
I wonder if that is that because it is an optional property in swagger? so the generated code thinks it could be there or not be there. If it is in fact guaranteed by the BE we can maybe fix the swagger to make it required?